pom.xml

```diff
@@ -731,6 +732,22 @@
                 <version>${dep.jackson.version}</version>
             </dependency>
 
+            <dependency>
```

Comment:
Can you please past the dependency tree and the jar size before and after this change ?

You can get this by doing a diff of the file before and after

mvn dependency:tree -pl presto-common

Reply:
Dependency tree: [before](https://gist.githubusercontent.com/sviscaino/f18b1a9bd368078a86faa99bfa240bf4/raw/e1494a2962598b5048d431ba16e77e3ca1cf7aea/mvn%2520dependency:tree%2520(on%2520master)), [after](https://gist.githubusercontent.com/sviscaino/f18b1a9bd368078a86faa99bfa240bf4/raw/e1494a2962598b5048d431ba16e77e3ca1cf7aea/mvn%2520dependency:tree%2520(on%2520sviscaino:add-jsonpath-jayway)), [diff](https://gist.githubusercontent.com/sviscaino/f18b1a9bd368078a86faa99bfa240bf4/raw/e1494a2962598b5048d431ba16e77e3ca1cf7aea/diff%2520before%2520after), ([view on diffchecker](https://www.diffchecker.com/alRssufs))

Reply:
File size comparison (after `mvn clean package -DskipTests` on both remote branch and master):

| File                                                    | Before    | After     | Δ       | Δ%     |
|---------------------------------------------------------|-----------|-----------|---------|--------|
| ./presto-common/target/presto-common-0.276-SNAPSHOT.jar | 576.54 KB | 577.57 KB | 1.04 KB | +0.18% |
| ./presto-jdbc/target/presto-jdbc-0.276-SNAPSHOT.jar     | 9.74 MB   | 9.74 MB   | 1.09 KB | +0.01% |
| ./presto-main/target/presto-main-0.276-SNAPSHOT.jar     | 10.07 MB  | 10.08 MB  | 6.78 KB | +0.07% |

All other jars are the same size.